Src/BootCamp.Chapter/Player.cs

```diff
@@ -0,0 +1,138 @@
+﻿using BootCamp.Chapter.Items;
+using System;
+using System.Net.NetworkInformation;
+using System.Reflection.Metadata;
+using System.Security.Cryptography.X509Certificates;
+
+namespace BootCamp.Chapter
+{
+    /// <summary>
+    /// Task: simulate a player who has an inventory.
+    /// Player can add or remove items from inventory.
+    ///
+    /// Extra task: player has equipement and maximum weight he/she can carry.
+    /// Player should not be able to take items if already at maximum weight.
+    /// Player should expose TotalAttack, TotalDefense stats.
+    /// </summary>
+    public class Player
+    {
+        /// <summary>
+        /// Everyone can carry this much weight at least.
+        /// </summary>
+        private const int baseCarryWeight = 30;
+        private const int baseHP = 60;
+        private readonly string _name;
+        public string Name { get { return _name; } }
+        private int _hp;
```

Comment:
Codacy found an issue: [Remove this unread private field '_hp' or refactor the code to use its value.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5776099)